external/avro/src/main/scala/org/apache/spark/sql/avro/AvroDeserializer.scala

```diff
@@ -1,427 +0,0 @@
-/*
```

Comment:
Hey, let's avoid renaming a file like this. It's difficult to track what was changed. Can we keep the class and file names?

Reply:
So previously, it was Decoder => GenericDatumReader => AvroDeserializer. And now it is Decoder => SparkAvroDatumReader. So in a sense it is a different reader but I can change the name to AvroDeserializer if it is better?